my_file.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+if [ -n "$1" ]; then
+num=$1
+else 
+num=50
+fi
+d=$(date +'%d_%m_%y')
+mkdir "zero_$(echo $d)_files"
+cd "zero_$(echo $d)_files"
+for i in $(seq 1 $num)
+do
+	for j in $(seq 1 $i)
```

Comment:
Simplify the loop using a single seq command, Instead of using nested seq commands, you can achieve the same result by using a single seq command with the start and end values separated by a comma. For example, $(seq -s '0' 1 $i) instead of the inner loop.